### PR TITLE
Add bouncing email cleaner lambda

### PR DESCRIPTION
**Pull request by mathieuhendey**

### CODE REVIEW

File: cloudwatch.tf [hunk 1]
```diff
@@ -272,3 +272,27 @@ resource "aws_cloudwatch_event_target" "MOTRWebHandler-WarmUpEventTarget" {
   rule      = "${aws_cloudwatch_event_rule.MOTR-WarmUpEventRule.name}"
   arn       = "${aws_lambda_alias.NPingerAlias.arn}"
 }
+###################################################################################################################################
+resource "aws_cloudwatch_log_group" "MotrBouncingEmailCleaner" {
```
Comment (Thomas-Franklin) [inline cloudwatch.tf]: Split a line after the } and before ##'s and give it a name such as the pinger above

File: iam.tf [hunk 2]
```diff
@@ -132,3 +132,27 @@ resource "aws_iam_role_policy" "LambdaWarmerRolePolicy" {
...
+    environment     = "${var.environment}"
+    account_id      = "${data.aws_caller_identity.current.account_id}"
+    lambda_function = "${aws_lambda_function.MotrBouncingEmailCleaner.function_name}"
+  }
+}
+
+resource "aws_iam_role_policy" "MotrBouncingEmailCleanerLambda" {
+  name   = "motr-subscr-loader-policy-${var.environment}"
+  role   = "${aws_iam_role.MotrSubscriptionLoaderLambda.id}"
+  policy = "${data.template_file.lambda_cleaner_permissions_policy.rendered}"
```
Comment (Thomas-Franklin) [inline iam.tf]: The role needs changed

Comment (Thomas-Franklin) [inline iam.tf]: And name

File: variables.tf [hunk 3]
```diff
@@ -278,6 +278,54 @@ variable "notifier_log_level" {
...
+}
+
+variable "MotrBouncingEmailCleaner_publish" {
+  type    = "string"
+  default = false
+}
+
+variable "MotrBouncingEmailCleaner_mem_size" {
+  type    = "string"
+  default = "768"
```
Comment (sebolabs) [inline variables.tf]: what are these defaults doing here? it's a module and it shouldn't have any defaults, please refer to other variables for other lambdas and see how it looks there

File: variables.tf [hunk 4]
```diff
@@ -278,6 +279,54 @@ variable "notifier_log_level" {
   description = "MotrNotifier Lambda log level"
 }
 
+variable "MotrBouncingEmailCleaner_s3_key" {
+  type = "string"
+}
```
Comment (sebolabs) [inline variables.tf]: description - see other similar variables

File: variables.tf [hunk 5]
```diff
@@ -278,6 +279,54 @@ variable "notifier_log_level" {
...
+
+variable "MotrBouncingEmailCleaner_ver" {
+  type = "string"
+}
```
Comment (sebolabs) [inline variables.tf]: description - see other similar variables

**Pull request merged**